application/config/migration.php

```diff
@@ -1,6 +1,6 @@
 <?php defined('BASEPATH') OR exit('No direct script access allowed');
 
 $config['migration_enabled'] = TRUE;
-$config['migration_type'] = 'sequential';
-$config['migration_version'] = 0;
+$config['migration_type'] = 'timestamp';
+$config['migration_version'] = 20240516002600;
```

Comment:
Let's stick to `sequential`, and since https://github.com/LibriVox/librivox-catalog/pull/239 has merged, this should now be `2` :)